crates/pallet-balanced-currency-swap-bridges-initializer/src/tests.rs

```diff
@@ -411,7 +411,7 @@ fn runtime_upgrade() {
             assert_eq!(<LastForceRebalanceAskCounter<v1::Test>>::get(), 0);
 
             // Do runtime upgrade hook.
-            v1::AllPalletsWithoutSystem::on_runtime_upgrade();
+            <(UpgradeInit<v1::Test>, v1::AllPalletsWithoutSystem)>::on_runtime_upgrade();
```

Comment:
Why not the `execute_update_init()`?

Reply:
Reimplemented with `execute_on_runtime_upgrade`